libs/feature/router/src/lib/default/state/query-params.utils.ts

```diff
@@ -42,8 +44,8 @@ export function expandQueryParams(
       } else if (isDateUrl(value)) {
         const [start, end] = value.split('..')
         expanded[key] = {
-          ...(start && { start: new Date(start) }),
-          ...(end && { end: new Date(end) }),
+          ...(start && { start: new Date(`${start}T00:00:00`) }),
```

Comment:
I guess `T00:00:00` stands for the time... Is the time necessary here?

Reply:
Yes, good point, the time is necessary here to prevent timezone issues. Actually, the URL does indeed not include the time as it is not needed, but as a `Date` in the app has a time it's set to midnight when it's read from the URL. Then, within the app, the date is handled in the local time of the browser and when it's written to the browser it's "cut" again. If we don't handle the time within the app, there are conflicts between the date chosen in the datepicker and the one encoded in the URL and sent to elasticsearch. 

You can test it by switching the timezone of your browser. I discovered the issue since cypress is not running in the same timezone in the CI as my machine.